deploy/docs/Best_Practices.md

```diff
@@ -1448,3 +1449,23 @@ kube-prometheus-stack:
     prometheusSpec:
       thanos: null
 ```
+
+## Parsing log content as json
+
+In order to parse and store log content as json following configuration has to be applied.
+This is needed due to correct metadata enrichment otherwise everything that is part of
+metadata like `kubernetes` tag will be visible toghether with log content instead of part
+of fields
```

Comment:
I would skip this. It is incomplete or you have to have context to understand the motivation

Reply:
> I would skip this. It is incomplete or you have to have context to understand the motivation

ok, just wanted to add at least some context